WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1150,6 +1166,19 @@ InviteStatusTentative       = background-color: @InviteStatusTentativeBgColor@;
 InviteStatusCounterBgColor  = #fff4d5
 InviteStatusCounter         = background-color: @InviteStatusCounterBgColor@; margin: @MailViewGutterSpace@ 0;
 InvCounter                  = padding:0 24px;
+ComposeViewContainer        = @PageBg@ @FullWidth@ @Conv2Bg@; box-sizing: border-box; padding: @MailViewGutterSpace@; padding-top: 0;
+ComposeViewTopHeader        = background-color: @AppC@;
+ComposeViewFromWrapper      = @Conv2Bg@
+ComposeViewToTd             = padding-top: @ComposeViewOutsidePadding@;
+ComposeViewSubjectInputField= border:none; color:@TxtC@; font-size:1.18rem; height:3rem; @FullWidth@ @BottomSeparator@
+ComposeViewHeaderRowFirstTd = padding-left: @ComposeViewOutsidePadding@;
+ComposeViewHeaderButtonTitle= text-align: left; padding: 0; display:inline; line-height: 3em;
+ComposeViewTinyMceContainer = 
+ComposeViewHtmlEditor       = padding:@ComposeViewOutsidePadding@; background-color: @AppC@; 
+ComposeViewHeaderRows       =
+ComposeViewHeaderEmptyRight = padding-right: 18px;
+ComposeViewButtonTitleHover = border-bottom: 1px dotted @hexToRGBA(PrimaryBrandingColor, 40)@; color: @PrimaryBrandingColor@;
```

Comment:
not at the widget level. We are using this style of ZButton hover classes that are in the application view (ZmComposeView in this case). So not targeting at the application level but most probably at all the view later. While working on other applications this could be improved or reused for New tasks/appointment/contacts based on the mockups.